tutorials/fileToJdbc/src/main/java/com/example/FileToJdbcApp/FileReader.java

```diff
@@ -0,0 +1,21 @@
+package com.example.FileToJdbcApp;
+
+import com.datatorrent.api.DefaultOutputPort;
+import org.apache.apex.malhar.lib.fs.LineByLineFileInputOperator;
+
+public class FileReader extends LineByLineFileInputOperator{
+
+  /**
+   * output in bytes to match CsvParser input type
+   */
+  public final transient DefaultOutputPort<byte[]> byteOutput  = new DefaultOutputPort<>();
+
+  @Override
+  protected void emit(String tuple)
+  {
+    output.emit(tuple);
+    byte[] bytes = tuple.getBytes();
+    byteOutput.emit(bytes);
+  }
```

Comment:
An entry needs to be added to the index in the parent directory (README.md) describing this application briefly.